dashboard/package.json

```diff
@@ -140,9 +140,24 @@
       "!src/**/*.d.ts"
     ]
   },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
   "husky": {
     "hooks": {
       "pre-commit": "yarn run ts-compile-check && lint-staged"
     }
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
```

Comment:
This looks interesting!? https://create-react-app.dev/docs/supported-browsers-features/#configuring-supported-browsers